internal/commands/server/command.go

```diff
@@ -71,6 +76,14 @@ func (c *Command) init() {
 	c.flagSet.IntVar(&c.flagSDSServerPort, "sds-server-port", defaultSDSServerPort, "SDS Server Port.")
 	c.flagSet.IntVar(&c.flagMetricsPort, "metrics-port", 0, "Metrics port, if not set, metrics are not enabled.")
 	c.flagSet.IntVar(&c.flagPprofPort, "pprof-port", 0, "Go pprof port, if not set, profiling is not enabled.")
+
+	{
+		// Consul namespaces
+		c.flagSet.StringVar(&c.flagConsulDestinationNamespace, "consul-destination-namespace", "", "Consul namespace to register gateway services.")
+		c.flagSet.BoolVar(&c.flagMirrorK8SNamespaces, "mirroring-k8s", false, "Register Consul gateway services based on Kubernetes namespace.")
+		c.flagSet.StringVar(&c.flagMirrorK8SNamespacePrefix, "mirroring-k8s-prefix", "", "Namespace prefix for Consul services when mirroring Kubernetes namespaces.")
```

Comment:
Assuming we don't expect these to conflict with any future flags by dropping the `-namespaces` suffix?

Reply:
Yeah, I don't imagine there are going to be any other flags regarding "mirroring", but the values I just dasherized the option names from helm, which are [mirroringK8S](https://www.consul.io/docs/k8s/helm#v-synccatalog-consulnamespaces-mirroringk8s) and [mirroringK8SPrefix](https://www.consul.io/docs/k8s/helm#v-synccatalog-consulnamespaces-mirroringk8sprefix).